test/helper-functions.php

```diff
@@ -2,28 +2,51 @@
 
 declare(strict_types = 1);
 
+use Rx\Observable;
+use Rx\Testing\MockObserver;
 use Rx\Testing\Recorded;
 use Rx\Testing\Subscription;
+use Rx\Functional\FunctionalTestCase;
 use Rx\Notification\OnCompletedNotification;
 use Rx\Notification\OnErrorNotification;
 use Rx\Notification\OnNextNotification;
 
-function onError(int $dueTime, $error, callable $comparer = null) {
+function onError(int $dueTime, $error, callable $comparer = null)
+{
     return new Recorded($dueTime, new OnErrorNotification($error), $comparer);
 }
 
-function onNext(int $dueTime, $value, callable $comparer = null) {
+function onNext(int $dueTime, $value, callable $comparer = null)
+{
     return new Recorded($dueTime, new OnNextNotification($value), $comparer);
 }
 
-function onCompleted(int $dueTime, callable $comparer = null) {
+function onCompleted(int $dueTime, callable $comparer = null)
+{
     return new Recorded($dueTime, new OnCompletedNotification(), $comparer);
 }
 
-function subscribe(int $start, int $end = PHP_INT_MAX) {
+function subscribe(int $start, int $end = PHP_INT_MAX)
+{
     return new Subscription($start, $end);
 }
 
-function RxIdentity($x) {
+function RxIdentity($x)
+{
     return $x;
 }
+
+function materializeObservable(Observable $observable): array
```

Comment:
@asm89 ACK I'll refactor it in a bit.  Aren't we planning to move some of the test helpers out of the testing namespace?